raiwidgets/raiwidgets/error_analysis_dashboard.py

```diff
@@ -121,6 +156,11 @@ def __init__(self, explanation=None, model=None, *, dataset=None,
             alternative to the model and explanation for a more limited
             view.
         :type pred_y: numpy.ndarray or list[]
+        :param pred_y_dataset: The predicted labels for the provided dataset.
```

Comment:
If we have `dataset` and `sample_dataset` should we also then have `y_pred` and `sample_y_pred` for consistency?

Reply:
We have true_y and true_y_dataset already, and this is following this with pred_y and pred_y_dataset.
This naming makes sense because in the common case a downsampled explanation is passed, we need the pred_y/true_y to follow the explanation but the pred_y_dataset/true_y_dataset to follow the dataset parameter.  However in the case there is no explanation passed, and user only passes in a dataset, we give the user the option to still pass in a sample_dataset for the dataset explorer only (similar to the dataset that is on the downsampled explanation).  So I think the current naming is best considering all of the possible different scenarios.